### PR TITLE
dev-cpp/cpp-taskflow: Bump to 3.0.0 

### PR DESCRIPTION
**Pull request by jsmolic**

### CODE REVIEW

File: dev-cpp/cpp-taskflow/cpp-taskflow-3.0.0.ebuild [hunk 1]
```diff
@@ -12,42 +12,30 @@ SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
...
 }
 
 src_install() {
-	cmake-utils_src_install
-
-	if $(use doc); then
-		einstalldocs
-	fi
+	cmake_src_install
+	use doc && einstalldocs
```
Comment (SoapGentoo) [inline dev-cpp/cpp-taskflow/cpp-taskflow-3.0.0.ebuild:40]: this is dead code, as `cmake_src_install` calls `einstalldocs` internally

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: dev-cpp/cpp-taskflow/cpp-taskflow-3.0.0.ebuild [hunk 2]
```diff
@@ -12,42 +12,30 @@ SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="doc"
-
-RDEPEND=""
-DEPEND=""
+IUSE="examples doc test"
```
Comment (SoapGentoo) [inline dev-cpp/cpp-taskflow/cpp-taskflow-3.0.0.ebuild:15]: where is `USE=test` used in the ebuild?

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: dev-cpp/cpp-taskflow/cpp-taskflow-3.0.0.ebuild (hunk 2 above)
Comment (jsmolic) [reply dev-cpp/cpp-taskflow/cpp-taskflow-3.0.0.ebuild]: It got lost during the process :)

File: dev-cpp/cpp-taskflow/cpp-taskflow-3.0.0.ebuild (hunk 1 above)
Comment (jsmolic) [reply dev-cpp/cpp-taskflow/cpp-taskflow-3.0.0.ebuild]: Ah that's right. Removed

File: dev-cpp/cpp-taskflow/cpp-taskflow-3.0.0.ebuild [hunk 3]
```diff
@@ -12,42 +12,28 @@ SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
...
 
 src_prepare() {
-	default
-
 	# fix library directoy
 	sed -i "s#/lib#/$(get_libdir)#g" CMakeLists.txt || die "sed failed"
 
-	cmake-utils_src_prepare
+	cmake_src_prepare
 }
```
Comment (SoapGentoo) [inline dev-cpp/cpp-taskflow/cpp-taskflow-3.0.0.ebuild:28]: do you still need to override this whole phase function?

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: dev-cpp/cpp-taskflow/cpp-taskflow-3.0.0.ebuild (hunk 3 above)
Comment (jsmolic) [reply dev-cpp/cpp-taskflow/cpp-taskflow-3.0.0.ebuild:28]: Actually no since we patched this out, I'll remove it

Comment (jsmolic) [reply dev-cpp/cpp-taskflow/cpp-taskflow-3.0.0.ebuild]: Done

**Pull request closed without merging**